src/sidebar/components/Annotation/AnnotationTimestamps.tsx

```diff
@@ -7,14 +7,13 @@ import {
   formatDate,
 } from '../../util/time';
 
-/**
- * @typedef AnnotationTimestampsProps
- * @prop {string} annotationCreated
- * @prop {string} annotationUpdated
- * @prop {string} [annotationUrl]
- * @prop {boolean} [withEditedTimestamp] - Should a timestamp for when this
- *   annotation was last edited be rendered?
- */
+export type AnnotationTimestampProps = {
+  annotationCreated: string;
+  annotationUpdated: string;
+  annotationUrl?: string;
+  /** Should a timestamp for when this annotation was last updated be rendered? */
```

Comment:
The phrasing here is a bit awkward. Something like "Display a relative last-updated timestamp" might be better.